src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -200,16 +221,16 @@ public ImageProcessor(ImageClassificationTransformer transformer)
                 _imagePreprocessingRunner.AddOutputs(transformer._resizedImageTensorName);
             }
 
-            public Tensor ProcessImage(string path)
+            public Tensor ProcessImage(VBuffer<byte> imageBuffer)
```

Comment:
(minor) we usually use `in VBuffer<byte> imageBuffer` parameters when dealing with VBuffers. This has a slight perf improvement, but even more important it signals that this parameter is "readonly" and no one will try to edit it. Sort of like `const` in C++.

Reply:
Will change.